random_joke.py

```diff
@@ -0,0 +1,30 @@
+#Flask app
+from flask import Flask
+import requests
+import json
+
+
+
+app = Flask(__name__)
+
+url = 'https://official-joke-api.appspot.com/random_joke'
+
+
+
+@app.route("/")
+def hello_world():
+    return "<h1>My first flask app!</h1>"
+
+@app.route("/random-joke")
+def return_joke():
+    response = requests.get(url)
+
+    #return api data as dictionary
+    api_dict = json.loads(response.text)
```

Comment:
When using `requests` you can simply do `api_dict = response.json()`

Reply:
Thank you.